server/api/member.go

```diff
@@ -64,7 +64,12 @@ func getMembers(svr *server.Server) (*pdpb.GetMembersResponse, error) {
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
+	dclocationDistribution, err := svr.GetTSOAllocatorManager().GetClusterDCLocationsFromEtcd()
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
 	for _, m := range members.GetMembers() {
+		m.DcLocation = "global"
```

Comment:
Does it mean this filed is `global` by default? What if we have a PD that doesn't configure `dc-location`? Should it be `global` also?

Reply:
Yes.

Reply:
How about empty if not config the zone label?

Reply:
"Global" is a concept in the tso allocator, and has no strong correlation with which DC localted.

Reply:
good point.